fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
There is an effort that was started around http://pagure.io/libpagure it's likely the closest thing to something official, but I do not know how updated it is.